peachjam/tasks.py

```diff
@@ -138,13 +138,40 @@ def extract_citations(document_id):
     try:
         if doc.extract_citations():
             doc.save()
+
+        extract_citation_contexts(document_id)
```

Comment:
this is already a background task -- can't we just extract the citation contexts in the same task? It's not a very complex operation and doesn't call remote APIs, so should be pretty quick.